client/my-sites/importer/uploading-pane.jsx

```diff
@@ -145,7 +145,9 @@ class UploadingPane extends React.PureComponent {
 					onClick={ isReadyForImport ? this.openFileSelector : null }
 					onKeyPress={ isReadyForImport ? this.handleKeyPress : null }
 				>
-					<div className="importer__upload-content">
+					<div
+						className={ `importer__upload-content ${ this.props.importerStatus.importerState }` }
```

Comment:
When combining classnames from a prop like this, you can use the `classnames` method. Rough example:

```
/**
 * External dependencies
 */
import classnames from 'classnames';

const classNames = classnames( 'importer__upload-content', this.props.importerStatus.importerState );

<div className={ classNames } />
```

A live example:
https://github.com/Automattic/wp-calypso/blob/ddb2a6a536d200635affce452094bd38fe571a02/client/reader/following-manage/feed-search-results.jsx#L58-L60